plagiarismFinder/src/test/scala/de/htwberlin/f4/wikiplag/plagiarism/MatchTest.scala

```diff
@@ -0,0 +1,166 @@
+package de.htwberlin.f4.wikiplag.plagiarism
+
+/**
+  * @author
+  * Kirill P.
+  *
+  **/
+
+import java.lang.Exception
+
+import de.htwberlin.f4.wikiplag.plagiarism.models.HyperParameters
+import org.junit.Test
+import org.scalatest.FunSuite
+import de.htwberlin.f4.wikiplag.plagiarism.models.HyperParameters
+import de.htwberlin.f4.wikiplag.utils.CassandraParameters
+import de.htwberlin.f4.wikiplag.utils.database.CassandraClient
+import junit.framework.Assert
+import junit.framework.Assert._
+import org.apache.spark.SparkContext
+import org.junit.{After, Before, Test}
+import org.scalatest.junit.AssertionsForJUnit
+import org.hamcrest.CoreMatchers
+import org.junit.Assert
+import org.scalactic.Fail
+import jdk.nashorn.internal.ir.annotations.Ignore
+
+import scala.util.control.Exception
+
+
+class MatchTest extends AssertionsForJUnit {
+
+  var finder: PlagiarismFinder = _
+  var sc: SparkContext = _
+
+  @Before def setUp() {
```

Comment:
better create spark context only once and not per test - expensive operation, takes a lot of time